platform_versioned_docs/version-24.1/enterprise/kubernetes-minimal-install.mdx

```diff
@@ -0,0 +1,418 @@
+---
+title: Install Seqera Platform on Kubernetes
+description: Deploy Seqera Platform Enterprise with Kubernetes
+date: "11 Sep 2024"
+tags: [kubernetes, deployment]
+---
+
+This installation guide describes how to install Seqera Platform on a Kubernetes cluster.
+
+## Prerequisites
+
+The following prerequisites are required to deploy Seqera on a Kubernetes cluster.
+
+- A Kubernetes cluster version 1.30.2 or newer
+- The `kubectl` CLI is installed locally
+- Access to the cluster with the `cluster-admin` role
+
+## 1. Create a namespace
+
+Create a namespace to isolate Kubernetes resources used by Seqera Platform from the other resources on your cluster.
+
+:::tip
+This installation guide assumes the use of `seqera-platform` as the installation namespace. Consider using a different one that better fits your cluster naming convention.
+:::
+
+1. Create a namespace for the Seqera resources:
+
+    ```
+    kubectl create namespace seqera-platform
+    ```
+
+    <details>
+    <summary>View command output</summary>
+    ```
+    namespace/seqera-platform created
+    ```
+    </details>
+
+1. Switch to the namespace:
+
+    ```
+    kubectl config set-context --current --namespace=seqera-platform
+    ```
+
+## 2. Configure container registry credentials
+
+Seqera Enterprise is distributed as a collection of Docker containers available through the Seqera container registry [`cr.seqera.io`](https://cr.seqera.io). Contact [support](https://support.seqera.io) to get your container access credentials. After you've received your credentials, grant your cluster access to the registry:
+
+1. Retrieve the `name` and `secret` values from the JSON file that you received from Seqera support.
+
+1. Create a [secret][kubectl-secret] for the image pull secret:
+
+    ```
+    kubectl create secret docker-registry cr.seqera.io \
+      --docker-server=cr.seqera.io \
+      --docker-username='<name>' \
+      --docker-password='<secret>'
+    ```
+
+    The credential `name` contains a dollar `$` character. Wrap the name in single quotes to prevent the Linux shell from interpreting this value as an environment variable.
```

Comment:
General comment there is a potential here to do an advanced admin guide and make a recommendation to download and host a copy of the containers yourself which removes a dependancy on us and makes their platform more resilient  and also removes the need for this secret to be maintained if they're using AWS ECR for example.